scripts/check_locales.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+
+work_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# List of keys that are missing, which is expected to be missing
+EXPECTED_MISSING=(
+    -e problems.show.languages.en
+    -e problems.show.languages.ru
+    -e problems.show.languages.uk
+    -e activerecord.attributes.user.username
+)
+
+LOCALES_CMD=(bundle exec i18n-tasks missing)
```

Comment:
`LOCALES_CMD` could be renamed to `MISSING_KEYS_CMD` to make its purpose more clear.